src/nvim/ex_docmd.c

```diff
@@ -6861,7 +6905,26 @@ void ex_cd(exarg_T *eap)
     if (new_dir == NULL || vim_chdir(new_dir))
       EMSG(_(e_failed));
     else {
-      post_chdir(eap->cmdidx == CMD_lcd || eap->cmdidx == CMD_lchdir);
+      int scope = 0; // Depends on what command was invoked
```

Comment:
Could be initialized to use SCOPE_EDITOR, and move 6911-6914 to the `default:` clause.